src/NuGet.Core/NuGet.Build.Tasks/NuGet.props

```diff
@@ -0,0 +1,31 @@
+<!--
+***********************************************************************************************
+NuGet.props
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (C) Microsoft Corporation. All rights reserved.
```

Comment:
I know this is the correct one to use today, see https://github.com/NuGet/Home/issues/6518, but NuGet.targets is .NET Foundation. 

https://github.com/NuGet/NuGet.Client/blob/a746cd6a01dfd22187375ee9c5a0d9b98676020a/src/NuGet.Core/NuGet.Build.Tasks/NuGet.targets#L9

I am not blocking this issue on thsi, just wanted to make sure @karann-msft see this.